api/v3/Job/Iatsverify.php

```diff
@@ -224,6 +224,8 @@ function civicrm_api3_job_iatsverify($params) {
       1 => count($error_log),
     )
   );
+  $message .= '<br />' . ts('Contribution ID: ') . !empty($contribution['id'])?? $contribution['id']; 
+  $message .= '<br />' . ts('Recurring Contribution ID: ') . !empty($contribution['contribution_recur_id'])?? $contribution['contribution_recur_id']; 
```

Comment:
This isn't the right place for this, you want it further up (this code will execute after looping through all the contributions, so at best you'd only see the last one).